packages/snaps-rpc-methods/src/restricted/dialog.test.ts

```diff
@@ -58,7 +62,64 @@ describe('implementation', () => {
     expect(hooks.showDialog).toHaveBeenCalledWith(
       'foo',
       DialogType.Alert,
-      panel([heading('foo'), text('bar')]),
+      'bar',
+      undefined,
+    );
+  });
+
+  it('gets the interface data if an interface ID is passed', async () => {
+    const content = text('foo');
+
+    const hooks = {
+      showDialog: jest.fn(),
+      isOnPhishingList: jest.fn(),
```

Comment:
Can we assert that this is called?

Reply:
It's done in a test further down in the file